TradeItIosTicketSDK2/TradeItViewController.swift

```diff
@@ -29,4 +29,42 @@ class TradeItViewController: CloseableViewController {
             containerView.bottomAnchor.constraint(equalTo: self.view.bottomAnchor)
         ])
     }
+
+    func fireViewEventNotification(view: TradeItNotification.View, title: String? = nil) {
```

Comment:
If Y! isn't using the Welcome screen then we should skip adding it there for now. We can add it in easy enough later on but I don't want to add it as a feature to all of the TradeIt screens if it's still being figured out.